apiserver/backupoptions/backupoptionsutil.go

```diff
@@ -98,7 +98,10 @@ func convertBackupOptsToStruct(backupOpts string, request interface{}) (backupOp
 
 	err = commandLine.Parse(parsedBackupOpts)
 	if err != nil {
-		return nil, nil, handleCustomParseErrors(err, usage, optsStruct)
+		err = handleCustomParseErrors(err, usage, optsStruct)
+		if err != nil {
```

Comment:
The use of `err` twice here is confusing. Which error is being passed to the function below?

Reply:
Sure i'll update it to be more clear

Reply:
Ill plan on updating to `customErr` unless i hear otherwise

Reply:
This can also use the

```go
if err := ...; err != nil {
}
``` syntax

Reply:
alright ill update with that